packages/web-components/src/components/carousel/__stories__/carousel.stories.scss

```diff
@@ -26,6 +26,10 @@
 
   /* stylelint-disable-next-line selector-type-no-unknown */
   dds-carousel {
+    /* The unit is required to correctly calculate the margin/padding for page overflow UI */
+    /* stylelint-disable-next-line length-zero-no-unit */
+    --dds--carousel--overflow-hint-size: 0rem;
```

Comment:
Wondering if this should be non-zero for larger breakpoints; This dictates the size of area showing "next card", and such area is required for larger breakpoints AFAIK.

Reply:
You are 100% right like always @asudoh :) Just fixed it